rplugin/python3/molten/moltenbuffer.py

```diff
@@ -395,7 +396,9 @@ def _show_selected(self, span: CodeCell) -> None:
             )
 
         if self.should_show_floating_win:
-            self.outputs[span].show_floating_win(span.end)
+            offset = span.calculate_output_offset()
+            self.outputs[span].show_floating_win(span.end, offset)
```

Comment:
`show_floating_win` is also called from `:MoltenShowOutput` and `:MoltenEnterOutput`.

Those don't get called with offsets, so the offset is only displayed properly right after the cell is run.

Reply:
I moved it into `show_floating_win` and `show_virtual_output`, so no need to add additional calls.